<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,10">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="speed">
          <property name="text">
           <string>速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_speed">
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>5000</number>
          </property>
          <property name="sliderPosition">
           <number>5000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="radius">
          <property name="text">
           <string>半径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_radius">
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>5000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="icon">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/background/images/stop.jpg);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
